### PR TITLE
mitosheet_private: create new mitosheet_private package

### PR DESCRIPTION
**Pull request by aarondr77**
# Description

Creates a new mitosheet-private python package that is just a wrapper around mitosheet. When mitohseet_private is installed, the mitosheet package is uses Pro. 
 
# Testing

- Create an old mitosheet-private deployment. Then move to this and make sure the analysis still works. 

# Documentation

Note if any new documentation needs to addressed or reviewed.

### CODE REVIEW

File: mitosheet-helpers/mitosheet_private/setup.cfg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[metadata]
```
Comment (naterush) [inline mitosheet-helpers/mitosheet_private/setup.cfg:1]: I think we can just use a setup.cfg, a no setup.py. Setup.py is actually in the process of being deprecated (and it's more confusing than a setup.cfg anyways). 

Wanna move over to setup.cfg?

File: mitosheet/mitosheet/user/utils.py [hunk 2]
```diff
@@ -32,6 +32,12 @@
 except ImportError:
     MITOSHEET_HELPER_ENTERPRISE = False
 
+try:
```
Comment (naterush) [inline mitosheet/mitosheet/user/utils.py:35]: I think there might be other checks of mitosheet private. I'll investigate now...

Comment (naterush) [inline mitosheet/mitosheet/user/utils.py:35]: I think there might be other checks of mitosheet private. I'll investigate now...

File: mitosheet-helpers/mitosheet_private/setup.cfg (hunk 1 above)
Comment (naterush) [reply mitosheet-helpers/mitosheet_private/setup.cfg:1]: You probably can post the setup.py in chat gpt and it'll likely nail it :)

**Review by naterush — COMMENTED** [review]
Nice! Glad you got to setup the package structure. Try moving it to `setup.cfg` for future proofing and simplicity, and then ping me and I can merge.

File: mitosheet/mitosheet/user/utils.py (hunk 2 above)
Comment (aarondr77) [reply mitosheet/mitosheet/user/utils.py:35]: Don't see anything else

**Pull request merged**